config/500-controller.yaml

```diff
@@ -66,6 +66,8 @@ spec:
           value: config-logging
         - name: CONFIG_OBSERVABILITY_NAME
           value: config-observability
+        - name: ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID
```

Comment:
Just note, S-O downloads the manifest files from knative-sandbox so the backport for the yaml does not work atm.
https://github.com/openshift-knative/serverless-operator/blob/main/openshift-knative-operator/hack/update-manifests.sh#L64

Reply:
Thanks for noting this I will patch it I guess as usual unless by that time we have migrated to fetching manifests from this repo.

Reply:
> Thanks for noting this I will patch it I guess as usual unless by that time we have migrated to fetching manifests from this repo.

Yeah, I think we can change the S-O's [update-manifests.sh](https://github.com/openshift-knative/serverless-operator/blob/main/openshift-knative-operator/hack/update-manifests.sh#L64) downloads the manifests from this repo (openshift-knative) instead of knative-sandbox.

I tried to think why it is using knative-sandbox repo currently but cannot find the reason.